KarhooSDK/Api/Response/VehicleImageRules.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  VehicleRules.swift
+//  KarhooSDK
+//
+//  Created by Aleksander Wedrychowski on 19/07/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+public struct VehicleImageRules: Codable, KarhooCodableModel {
+    public let rules: [VehicleImageRule]
+
+    enum CodingKeys: String, CodingKey {
+        case rules = "mappings"
+    }
+}
+
+public struct VehicleImageRule: Codable {
+    
+    private static let fallbackRuleValue = "*"
+
+    public enum RuleType {
+        /// Rule that should be used in a first place. Vehicle should mach type and it's tags list should be same as tags enlisted in a rule.
```

Comment:
you have some typos in the description. took the opportunity to rephrase it for clarity: `Rule that should be used when the vehicle type and tags match perfectly the ones in the quote.`